app/src/main/res/values/strings.xml

```diff
@@ -41,10 +41,16 @@
     <string name="back">Back</string>
     <string name="username">Username</string>
     <string name="log_out">Log Out</string>
-    
-    
+
+    <!-- Bundle keys -->
+    <string name="post_title_key">post_title_key</string>
+    <string name="post_content_key">post_content_key</string>
+
     <!-- Shared preferences keys -->
     <string name="pref_notifications">pref_notifications</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Why is this string value added back in? it is not used and has been removed previously.